sample/src/main/res/layout/activity_sample.xml

```diff
@@ -1,69 +1,80 @@
-        android:layout_height="match_parent"
-        android:background="#FFAAAAAA"
-        tools:context=".SampleActivity">
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#FFAAAAAA"
+    tools:context=".SampleActivity">
 
     <com.willowtreeapps.signinwithapplebutton.view.SignInWithAppleButton
-            android:id="@+id/sign_in_with_apple_button_black"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            style="@style/SignInWithAppleButton.Black"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintBottom_toTopOf="@+id/sign_in_with_apple_button_white"
-            android:layout_marginStart="20dp" android:layout_marginEnd="20dp"/>
+        android:id="@+id/sign_in_with_apple_button_black"
+        style="@style/SignInWithAppleButton.Black"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginEnd="20dp"
+        app:layout_constraintBottom_toTopOf="@+id/sign_in_with_apple_button_white"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
     <com.willowtreeapps.signinwithapplebutton.view.SignInWithAppleButton
-            android:id="@+id/sign_in_with_apple_button_white"
-            android:layout_height="30dp"
-            style="@style/SignInWithAppleButton.White"
-            app:sign_in_with_apple_button_textType="signInWithApple"
-            app:sign_in_with_apple_button_cornerRadius="0dp"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/sign_in_with_apple_button_black"
-            app:layout_constraintBottom_toTopOf="@+id/sign_in_with_apple_button_white_outline"
-            android:layout_width="wrap_content"/>
+        android:id="@+id/sign_in_with_apple_button_white"
+        style="@style/SignInWithAppleButton.White"
+        android:layout_width="wrap_content"
+        android:layout_height="30dp"
+        app:layout_constraintBottom_toTopOf="@+id/sign_in_with_apple_button_white_outline"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/sign_in_with_apple_button_black"
+        app:sign_in_with_apple_button_cornerRadius="0dp"
+        app:sign_in_with_apple_button_textType="signInWithApple" />
```

Comment:
If it's all the same, I'd like to keep the three customization points ordered the same as in the README, and grouped together where they're used.

Reply:
Bah, I just ran the auto-formatter on it.